template/composer.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "zendframework/<project-name>",
+    "description": "<project-description>",
+    "homepage": "https://docs.zendframework.com/<project-name>/",
+    "type": "library",
+    "license": "BSD-3-Clause",
+    "keywords": [
+        "http",
+        "middleware",
+        "psr",
+        "psr-7",
+        "psr-11"
+    ],
+    "support": {
+        "docs": "https://docs.zendframework.com/<project-name>/",
+        "issues": "https://github.com/zendframework/<project-name>/issues",
+        "source": "https://github.com/zendframework/<project-name>",
+        "slack": "https://zendframework-slack.herokuapp.com",
+        "forum": "https://discourse.zendframework.com"
```

Comment:
Should we have here link to the main page of forum or maybe specific category there?
https://github.com/zendframework/zend-expressive/blob/master/composer.json#L19